### PR TITLE
Add foreground window check.yml

### PR DESCRIPTION
**Pull request by ejfocampo**

### CODE REVIEW

**Comment by ejfocampo** [comment]
Initial checks failed. Local testing shows no problem with linting.  
```
python3 ../scripts/lint.py --thorough -t "foreground window check" -v . 
INFO:lint:successfully loaded 829 rules
INFO:lint:collecting potentially referenced samples
INFO:lint:no lints failed, nice! 
```
I might've requested the PR too early. Probably need to merge https://github.com/mandiant/capa-testfiles/pull/209 first.

**Comment by mr-tz** [comment]
Yes, that should be fixed once the testfiles PR is merged.

File: anti-analysis/anti-av/foreground-window-check.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+rule:
+  meta:
+    name: foreground window check
+    namespace: anti-analysis/anti-av
```
Comment (williballenthin) [inline anti-analysis/anti-av/foreground-window-check.yml:4]:
```suggestion
    namespace: anti-analysis/anti-vm/vm-detection
```

??

**Review by williballenthin — APPROVED** [review]

File: anti-analysis/anti-av/foreground-window-check.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+rule:
+  meta:
+    name: foreground window check
```
[the remark this answers is not in the document]
Comment (ejfocampo) [reply anti-analysis/anti-av/foreground-window-check.yml:3]: adjusted. thanks for the feedback

File: anti-analysis/anti-av/foreground-window-check.yml
Comment (mr-tz) [inline anti-analysis/anti-av/foreground-window-check.yml:1]: can you remove this file and then we're good to merge

Comment (ejfocampo) [reply anti-analysis/anti-av/foreground-window-check.yml:1]: ooops. deleted. thanks for the catch Moritz.

**Review by mr-tz — APPROVED** [review]
thanks!

**Pull request merged**